package.json

```diff
@@ -256,33 +256,34 @@
         "vscode:prepublish": "npm run compile",
         "compile": "tsc -p ./",
         "watch": "tsc -watch -p ./",
-        "postinstall": "node ./node_modules/vscode/bin/install",
```

Comment:
Any idea what the "postinstall" script was doing and why we don't need it anymore?

Reply:
Let's remove it then and if something blows up, we'll figure it out then. Thanks for checking out what it was!